spec/requests/notifications_spec.rb

```diff
@@ -1,24 +1,23 @@
 # frozen_string_literal: true
 
-RSpec.describe NotificationsController, type: :controller do
+describe 'Notifications', type: :request do
```

Comment:
```suggestion
describe 'Notification', type: :request do
```

Nitpick: I noticed that some of the request _spec are plural and some are singular. I think for consistency, let's stick to singular to match the rest of the example request specs: https://relishapp.com/rspec/rspec-rails/docs/request-specs/request-spec . Same goes for filename.

Reply:
I've always done it to match the controllers/routing. So if it's `notifications_controller` or `GET /notifications`, I'd do `notifications_spec`. But there's also some inconsistent naming in the controllers too so 🤷‍♀️

Reply:
Have updated name of the spec to singular.